sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientTests.cs

```diff
@@ -1,177 +1,99 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using Azure.Core.Pipeline;
-using Azure.Core.Testing;
-using NUnit.Framework;
 using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text.Json;
-using System.Threading.Tasks;
+using System.Text;
+using Azure.Core;
+using Azure.Core.Testing;
+using Azure.Identity;
+using NUnit.Framework;
 
 namespace Azure.AI.TextAnalytics.Tests
 {
     public class TextAnalyticsClientTests : ClientTestBase
     {
-        private static readonly string s_endpoint = "https://contoso-textanalytics.cognitiveservices.azure.com/";
-        private static readonly string s_subscriptionKey = "FakeSubscriptionKey";
-
         public TextAnalyticsClientTests(bool isAsync) : base(isAsync)
         {
-        }
-
-        private TextAnalyticsClient CreateTestClient(HttpPipelineTransport transport)
-        {
-            var options = new TextAnalyticsClientOptions
+            TextAnalyticsClientOptions options = new TextAnalyticsClientOptions
             {
-                Transport = transport
+                Transport = new MockTransport(),
```

Comment:
It looks like this is the mock, and the one called mock is the live one... or am I misunderstanding?

Reply:
Neither actually call the service, so in a sense both are mocks. The idea was to separate them:
- the one called mock => actually is mocking the call and creating objects to compare against
- This class is only doing unit testing. We do need a client in order to access the methods but not actual/mocking calls are being done to those methods. 

If it is too confusing I can always merge them under the `Mock` one

Reply:
No, that's fine, it makes sense to me now.  Once we add TextAnalyticsClientLiveTests.cs, it will be clear.